core/controller/src/main/scala/whisk/core/entitlement/ActivationThrottler.scala

```diff
@@ -113,8 +100,8 @@ class ActivationThrottler(consulServer: String, concurrencyLimit: Int, systemOve
     }
 
     Scheduler.scheduleWaitAtLeast(healthCheckInterval) { () =>
+        val loadbalancerActivationCount = loadBalancer.getUserActivationCounts()
         for {
-            loadbalancerActivationCount <- getLoadBalancerActivationCount()
```

Comment:
Either write

```
for {
    loadbalancerActivationCount = loadBalancer.getUserActivationCounts()
    invokerActivationCount <- getInvokerActivationCount()
}
...
```

or we should get rid of the for comprehension in favor of just a "map". What do you think?